test/index.js

```diff
@@ -437,15 +437,17 @@ describe('views()', () => {
         const test = {
             name: 'test',
 
-            register: function (server, options) {
+            register: async function (server, options) {
 
                 server.path(__dirname);
 
                 const views = {
                     engines: { 'html': Handlebars },
-                    path: './templates/plugin'
+                    path: __dirname + '/templates/plugin'
```

Comment:
Why this change?

Reply:
This might've slipped through when trying to write tests based off of others. I can remove it!